ops/model.py

```diff
@@ -114,6 +114,7 @@
 logger = logging.getLogger(__name__)
 
 MAX_LOG_LINE_LEN = 131071  # Max length of strings to pass to subshell.
+MAX_RELATION_SET_LINE_LEN = 131000  # Max length of relation key-value data
```

Comment:
It appears this is unused?

Reply:
It is -- forgotten from a first pass. Removed now.